scip.proto

```diff
@@ -91,6 +91,15 @@ message Document {
   // but have a reference and are defined by some other symbol (see
   // Relationship.is_definition).
   repeated SymbolInformation symbols = 3;
+
+  // (optional) Text contents of the this document. Indexers are not expected to
+  // include the text by default. It's preferrable that clients read the text
+  // contents from the file system by resolving the absolute path from joining
+  // `Index.metadata.project_root` and `Document.relative_path`. This field was
+  // introduced to support `SymbolInformation.signature_documentation`, but it
+  // can be used for other purposes as well, for example testing or when working
+  // with virtual/in-memory documents.
+  string text = 5;
```

Comment:
what about "content" instead of "text"? don't have a strong preference either way.

Reply:
I went with `text` to mirror `TextDocumentItem.text` in LSP (https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocumentItem) and `Document.getText()` in VS Code. I'm personally fine either way but think it's a safe choice to stick with LSP terminology when appropriate.